lib/book.rb

```diff
@@ -0,0 +1,31 @@
+class Book
+  attr_accessor :read
```

Comment:
The attr_accessor isn't necessary since we never access the read attribute directly